Flow.Launcher.Core/Plugin/JsonRPCPluginSettings.cs

```diff
@@ -83,8 +83,8 @@ public void UpdateSettings(IReadOnlyDictionary<string, object> settings)
                             break;
                     }
                 }
-                Save();
             }
+            Save();
```

Comment:
sorry another thoughts that why didn't we keep this consistent with other plugins. `Save` only be called when manually triggered or flow being closed?

Reply:
I guess just personal preference. It feels off to not save to file on change.

Reply:
yeah ok